src/sagemaker_sklearn_extension/feature_extraction/sequences.py

```diff
@@ -393,30 +445,76 @@ def _hybrid_interpolation(x):
     def _extract_tsfresh_features(self, X):
         X_df = self._convert_to_df(X)
         X_df_no_nans = X_df.dropna()
```

Comment:
I think when X consists of all empty strings, then X_df has only nans and this leaves X_df_no_nans empty (len(X_df_no_nans) == 0). This will make tsfresh_features = extract_features(...) below will fail. We should add a small workaround to make the transformer robust to this corner case.

Reply:
Something along the lines of this:

```
        if len(X_df_no_nans) == 0:
            X_df_no_nans = X_df.loc[[0]].fillna(0)
```

Reply:
Is `X` here a single column or multiple columns. If it is multiple columns, then many values the are valid for a column could be discarded due to invalid in other columns

Reply:
X is a single column here. It's only multiple columns in the wrapper above called TSFeatureExtractor.

Reply:
fixed.